packages/neuron-ui/src/components/RecycleUDTCellDialog/index.tsx

```diff
@@ -0,0 +1,309 @@
+import React, { useCallback, useEffect, useState, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useState as useGlobalState } from 'states'
+import { sudtValueToAmount, shannonToCKBFormatter } from 'utils/formatters'
+import Dialog from 'widgets/Dialog'
+import AlertDialog from 'widgets/AlertDialog'
+import Button from 'widgets/Button'
+import TextField from 'widgets/TextField'
+import { isErrorWithI18n } from 'exceptions'
+import Hardware from 'widgets/Icons/Hardware.png'
+import Alert from 'widgets/Alert'
+import { useHardWallet, usePassword } from 'components/CellManagement/hooks'
+import {
+  getUDTTokenInfoAndBalance,
+  generateRecycleUDTCellTx,
+  openExternal,
+  sendTx,
+  signAndBroadcastTransaction,
+  OfflineSignStatus,
+  OfflineSignType,
+} from 'services/remote'
+import {
+  UDTType,
+  addressToScript,
+  isSuccessResponse,
+  isMainnet as isMainnetUtil,
+  validateAddress,
+  errorFormatter,
+  getExplorerUrl,
+} from 'utils'
+import styles from './recycleUDTCellDialog.module.scss'
+
+export interface DataProps {
+  address: string
```

Comment:
The address parameter is currently assigned to lockArgs, therefore renaming it to lockArgs is recommended for semantic accuracy.